r/R/utilities_expression.R

```diff
@@ -113,7 +119,7 @@ completeExpression <- function(expression, cell_ids) {
     on = .(cell_ids)
     ]
 
-  expression[, cell_ids := NULL]
```

Comment:
why this change? in-place deletion is much much faster. And all this expression code needs to be as fast as it can be (data exploration user experience currently depends heavily on performance here)

Reply:
Again, due to unintended side effects in parent function and because removal of a column is incredible fast (I would guess on order of nanoseconds). This is premature optimization -- other things will be slower in this function and so optimizing this will have no effect.